src/FSLibrary/StellarSupercluster.fs

```diff
@@ -27,14 +28,40 @@ let ExpandHomeDirTilde (s: string) : string =
     else
         s
 
+type ProcResult =
+    | RanSuccess
+    | RanWithError of int
+    | DidNotRun
+
 // Loads a config file and builds a Kubernetes client object connected to the
 // cluster described by it. Takes an optional explicit namespace and returns a
 // resolved namespace, which will be taken from the config file if no explicit
 // namespace is provided.
 let ConnectToCluster (cfgFile: string) (nsOpt: string option) : (Kubernetes * string) =
     let cfgFileExpanded = ExpandHomeDirTilde cfgFile
+
+
+    let runKubectl (args: List<string>) : ProcResult =
+        let args = [ "--kubeconfig"; cfgFileExpanded ] @ args
+        LogInfo "Attempting to run `kubectl %s`" (System.String.Join(" ", args))
```

Comment:
I think this log line causes tokens to leak into the logs from the `set-credentials` calls. That feels a little sketchy to me, as I assume these tokens are at least somewhat private? That's not such a huge deal with local runs, but maybe something we want to avoid in Jenkins logs.